server/views/pages/create/caseload.njk

```diff
@@ -10,9 +10,13 @@
 {% set backLinkHref = "/" %}
 {% set noJsBackLink = true %}
 
-{% macro offenderName(offenderName, crn, prisonerNumber, index) %}
+{% macro offenderName(offenderName, crn, prisonerNumber, probationPractitioner, index) %}
     <div class="caseload-offender-name">
-        <button id="name-button-{{ index }}" type="submit" name="prisonerNumber" value='{{ prisonerNumber }}' class="link-button govuk-heading-s govuk-!-padding-left-0 govuk-!-margin-bottom-0">{{ offenderName }}</button>
+        {% if probationPractitioner %}
+            <button id="name-button-{{ index }}" type="submit" name="prisonerNumber" value='{{ prisonerNumber }}' class="link-button govuk-heading-s govuk-!-padding-left-0 govuk-!-margin-bottom-0">{{ offenderName }}</button>
+        {% else %}
+            <span id="name-button-{{ index }}" class="govuk-heading-s govuk-!-padding-left-0 govuk-!-margin-bottom-0">{{ offenderName }}</span>
```

Comment:
Disabled creating a licence for an unallocated case - probably need to come back to this with solid requirements, at the moment, responsible_com is a required DB field

Reply:
Ok for now. I suspect requirements will come to allow OMU users to create `default` licences, but not yet. We might need another type of link - ie. not a COM, but an OMU creator.